docker-bake.hcl

```
@@ -315,14 +390,26 @@ target "rhel_ubi9" {
     VERSION      = REMOTING_VERSION
     JAVA_VERSION = "${javaversion(jdk)}"
   }
-  tags      = linux_tags(type, jdk, "rhel-ubi9")
+  tags = [
+    # If there is a tag, add versioned tag suffixed by the jdk
+      equal(ON_TAG, "true") ?
+      "${REGISTRY}/${orgrepo(type)}:${REMOTING_VERSION}-${BUILD_NUMBER}-rhel-ubi9-jdk${jdk}${equal(jdk, 25) ? "-preview" : ""}"
+      : "",
+    # If there is a tag and if the jdk is the default one, add versioned short tag
+      equal(ON_TAG, "true") ? (is_default_jdk(jdk) ? "${REGISTRY}/${orgrepo(type)}:${REMOTING_VERSION}-${BUILD_NUMBER}-rhel-ubi9" : "") : "",
+    # If the jdk is the default one, add rhel and latest short tags
+      is_default_jdk(jdk) ? "${REGISTRY}/${orgrepo(type)}:rhel-ubi9" : "",
+      is_default_jdk(jdk) ? "${REGISTRY}/${orgrepo(type)}:latest-rhel-ubi9" : "",
+    "${REGISTRY}/${orgrepo(type)}:rhel-ubi9-jdk${jdk}${equal(jdk, 25) ? "-preview" : ""}",
+    "${REGISTRY}/${orgrepo(type)}:latest-rhel-ubi9-jdk${jdk}${equal(jdk, 25) ? "-preview" : ""}",
+  ]
   platforms = rhel_ubi9_platforms(jdk)
 }
 
 target "nanoserver" {
   matrix = {
-    type            = windowsagenttypes(WINDOWS_AGENT_TYPE_OVERRIDE)
-    jdk             = jdks_to_build
+    type                       = windowsagenttypes(WINDOWS_AGENT_TYPE_OVERRIDE)
+    jdk                         = jdks_to_build
```

```suggestion
    type            = windowsagenttypes(WINDOWS_AGENT_TYPE_OVERRIDE)
    jdk             = jdks_to_build
```
todo: restore indentation